converter.go

```diff
@@ -354,6 +354,74 @@ func decimalToBigFloat(num decimal128.Num, scale int64) *big.Float {
 	return new(big.Float).Quo(f, s)
 }
 
+// ArrowSnowflakeTimestampToTime converts original timestamp returned by Snowflake to time.TIme
+func ArrowSnowflakeTimestampToTime(rec arrow.Record, rb *ArrowBatch, colIdx int, recIdx int) *time.Time {
+	scale := int(rb.scd.RowSet.RowType[colIdx].Scale)
+	dbType := strings.ToUpper(rb.scd.RowSet.RowType[colIdx].Type)
```

Comment:
perhaps we can encapsulate scale, dbtype and loc in a new structure? that way we wouldn't have to pass the whole ArrowBatch to the method? Just an idea

Reply:
I don't think it is a good option. In such case a driver user will have to extract data from ArrowBatch by himself and create such object to pass. Moreover scd and loc aren't public.